src/server_utils/message_handlers.rs

```diff
@@ -75,10 +75,17 @@ pub async fn handle_client_messages(
             }
         }
         models::WSClientMessage::UpdateProgress { game_id, progress } => {
-            db.update_game_progress(&game_id, current_user_id, progress)
-                .await;
+            let current_progress = db.find_game_progress(&game_id, current_user_id).await;
+            
+            if progress < 70 && progress - current_progress < 5 {
+                eprintln!("Skipping update progress of game_id: {game_id}, current_user_id: {current_user_id}, progress: {progress}");
+            } else {
+                db.update_game_progress(&game_id, current_user_id, progress)
+                    .await;
```

Comment:
can we update the game progress every time, but broadcast it to all the other users only if the progress is significant ( the if condition ). Some updated might not be persisted in the case of partial updates

Reply:
This was done.

Reply:
This project has been moved to https://github.com/Narayanbhat166/blazer, however it is still under development. You can feel free to contribute. Let me know if you need any help. Will be closing this issue.